ui/components/ui/confusable/README.mdx

```diff
@@ -0,0 +1,23 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import Confusable from './confusable.component';
+
+# Confusable
+
+Confusable allows to filter out any characters that look very similar to English language characters.
+
+<Canvas>
+  <Story id="ui-components-ui-confusable-confusable-stories-js--default-confusable" />
+</Canvas>
+
+## Component API
+
+<ArgsTable of={Confusable} />
```

Comment:
Hmm this doesn't seem to be displaying properly?

<img width="1440" alt="Screen Shot 2022-07-13 at 1 04 36 PM" src="https://user-images.githubusercontent.com/8112138/178824727-04c54334-f463-43eb-96b0-467a129afbe8.png">


Sometimes the argsTable doesn't like how the component is built. If it's to big of a change to the component we can just use a static markdown table.
```suggestion
| Name  | Description | Default |
| ----- | ----------- | ------- |
| input | string      | -       |
```